static/app/components/issues/suspect/suspectTable.tsx

```diff
@@ -57,6 +57,7 @@ const GradientBox = styled('div')`
   height: max-content;
   display: flex;
   flex-direction: column;
+  gap: ${space(1)};
```

Comment:
lets try to use the theme vars:

```suggestion
  gap: ${p => p.theme.space.md};
```